src/log-analytics-solution/azext_log_analytics_solution/_help.py

```diff
@@ -0,0 +1,71 @@
+# coding=utf-8
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+# pylint: disable=too-many-lines
+# pylint: disable=line-too-long
+from knack.help_files import helps  # pylint: disable=unused-import
+
+
+helps['monitor log-analytics solution'] = """
+    type: group
+    short-summary: Commands to manage monitor log-analytics solution.
+"""
```

Comment:
No need to indent the help. <kbd>Ctrl</kbd>+<kbd>A</kbd> to select all, then press <kbd>Shift</kbd>+<kbd>Tab</kbd> to remove the indentations.

Reply:
Got it~